x/ccv/provider/keeper/grpc_query.go

```diff
@@ -266,11 +266,13 @@ func (k Keeper) QueryConsumerChainsValidatorHasToValidate(goCtx context.Context,
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// get all the consumer chains for which the validator is opted-in
+	// get all the consumer chains for which the validator is already or will be
+	// opted-in starting from the next epoch
```

Comment:
This comment doesn't match the code below, right?

Reply:
I think it does. When `k.IsOptedIn` returns `true`, it means that the validator is either already on the consumer valset or is opted-in and will be added to the set at the start of the next epoch.

Reply:
Hm, I guess you are right. I'm just a bit confused what should happen when someone has opted out, but is still currently in power on the consumer chain